app/controllers/api/v1/admin/shipments_controller.rb

```diff
@@ -1,4 +1,6 @@
-class Api::V1::ShipmentsController < ApplicationController
+class Api::V1::Admin::ShipmentsController < Api::V1::Admin::BaseController
```

Comment:
Admin controllers all inherit from the admin base controller. The base controller also inherits from application controller